app/models/animal.rb

```diff
@@ -1,2 +1,3 @@
 class Animal < ApplicationRecord
+    #has_many:sightings
```

Comment:
Normally I don't want commented code as part of the PR but seeing as you are about to use this I am gonna let it slide.